WalletWasabi.Gui/Controls/WalletExplorer/CoinListViewModel.cs

```diff
@@ -458,6 +475,7 @@ public void OnCoinIsSelectedChanged(CoinViewModel cvm)
 			SetSelections();
 			SelectionChanged?.Invoke(this, cvm);
 			TotalAmount = Coins.Where(x=>x.IsSelected).Sum(x=>x.Amount.ToDecimal(NBitcoin.MoneyUnit.BTC));
+			IsAnyCoinSelected = TotalAmount > 0m;
```

Comment:
Maybe using a converter to TotalAmount instead of `IsAnyCoinSelected` a better pattern?

Reply:
But I don't know. @molnard thoughts?

Reply:
Oh no, subscribe to TotalAmount to set this IsAnyCoinSelected.